ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -147,6 +147,54 @@
       }
     ]
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "CLAIM_ISSUE",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "[APPLICANTSOLICITORONE]",
+          "[CREATOR]",
```

Comment:
Why do we need [CREATOR] case role here? By then the case role should have been removed.